packages/schema/scripts/prepack.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import fs from "fs";
+import path from "path";
+import schema from "~/breadboard.schema.json";
```

Comment:
Whoa, what is `~` in an import path?

Reply:
haha just a path alias
https://github.com/breadboard-ai/breadboard/pull/538/files/6e744544fc382858ea8e62ab5570023289e3aea7#diff-fc763b82b8113516d39db871392775c27b0e8427331bd7f0add8c64d3101802bR6-R8

Reply:
Huh. What happens when you `tsc compile` this? Does tsc rewrite the specifier to something?

Why not just do `from "../breadboard.schema.json"`?

Reply:
tsc leaves paths alone. with a bundler they would be rewritten. I prefer it for two reasons. I think the absolute import is tidier. And no need to refactor absolute imports when moving a file.

Reply:
I prefer the relative import. It will work without ts-node and is the normal idiomatic way to refer to other files. I would rate those concerns higher than the others.

Reply:
reasonable. will update